samples/bluetooth/RSSI_power_control/peripheral/src/main.c

```diff
@@ -0,0 +1,99 @@
+#include <zephyr/bluetooth/addr.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+
+#define MIN_ADV_INT 800
```

Comment:
why not use 800 for both? Or simply use a default parameter set like `BT_LE_ADV_CONN_FAST_1` ?